examples/ephemeral_gpu_offloading/load_with_dora.py

```diff
@@ -0,0 +1,90 @@
+# Example script demonstrating the time difference loading a model with a DoRA using ephemeral GPU offloading
+# vs doing it purely on the CPU.
+
+import argparse
+import time
+
+from huggingface_hub import snapshot_download
+from transformers import AutoModelForCausalLM
+
+from peft import PeftModel
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Load a model with DoRA using ephemeral GPU offloading")
+    parser.add_argument("--model", type=str, default="NousResearch/Hermes-2-Pro-Mistral-7B", help="Model to load")
+    parser.add_argument(
+        "--dora",
+        type=str,
+        default="anamikac2708/Mistral-7B-DORA-finetuned-investopedia-Lora-Adapters",
+        help="DoRA to use",
+    )
+    parser.add_argument("--ephemeral_gpu_offload", action="store_true", help="Use ephemeral GPU offloading")
+    parser.add_argument(
+        "--merge_model", action="store_true", help="Merge the model with the DoRA model and save to disk"
```

Comment:
How about changing this argument from a bool to a path, so that users can directly indicate where to save the merged model. If nothing is passed, the model is not merged and saved.